solidity/random-beacon/contracts/RandomBeacon.sol

```diff
@@ -146,6 +146,11 @@ contract RandomBeacon is Ownable {
     ///         operator affected.
     uint256 public relayEntryTimeoutNotificationRewardMultiplier;
 
+    /// @notice This amount is required to protect against griefing the staking
+    ///         contract and individual applications are allowed to require higher
+    ///         minimum stakes if necessary.
+    uint96 public minimumStake;
```

Comment:
I think we should call it `minimumAuthorization` to be precise. `RandomBeacon`, from T staking contract perspective, is an application that will have to implement `authorizationIncreased`, `authorizationDecreaseRequested`, `involuntaryAuthorizationDecrease` [from `IApplication`](https://github.com/threshold-network/solidity-contracts/blob/main/contracts/staking/IApplication.sol). 

Also, from [T staking contract RFC](https://github.com/threshold-network/solidity-contracts/blob/main/docs/rfc-1-staking-contract.adoc#authorizing-an-application-1):

> The application being authorized is notified via a callback every single time new authorization happens and the application may decide to revert the transaction if individual requirements of the application are not met. For example, application may require a certain minimum authorized amount.

Reply:
Sure, done in 9b2d771